backend/src/exercise/action-wrapper.ts

```diff
@@ -20,7 +20,7 @@ export class ActionWrapper extends NormalType<
                 this.databaseService.actionWrapperService.getFindById(this.id!)(
                     manager
                 );
-            const getNew = () => new ActionWrapperEntity();
+            const getNew = ActionWrapperEntity.createNew;
```

Comment:
It *could* be that this function would run in an unexpected context. To test it out one could reference a class variable (like `this.actionString`) in `ActionWrapperEntity.createNew`. However, one would easily notice this when trying it out.

A way to fic it would be to do this (I think): 
```ts
const getNew = () => ActionWrapperEntity.createNew();
```